src/couch_views/src/couch_views_indexer.erl

```diff
@@ -407,17 +408,27 @@ fetch_docs(Db, Changes) ->
         }
     end, #{}, erlfdb:wait_for_all(RevFutures)),
 
+    AddLocalSeq = lists:keymember(<<"local_seq">>, 1, DesignOpts),
     BodyFutures = maps:keys(BodyState),
     ChangesWithDocs = lists:map(fun (BodyFuture) ->
         {Id, RevInfo, Change} = maps:get(BodyFuture, BodyState),
         Doc = fabric2_fdb:get_doc_body_wait(Db, Id, RevInfo, BodyFuture),
 
         BranchCount = maps:get(branch_count, RevInfo, 1),
-        Doc1 = if BranchCount == 1 -> Doc; true ->
-            RevConflicts = fabric2_fdb:get_all_revs(Db, Id),
-            {ok, DocWithConflicts} = fabric2_db:apply_open_doc_opts(Doc,
-                RevConflicts, [conflicts]),
-            DocWithConflicts
+        Doc1 = case BranchCount == 1 of
```

Comment:
I didn't see BranchCount used else, so perhaps it could just be removed and instead the clause can be 👍 
```
case maps:get(branch_count, RevInfo, 1) of
   1  when AddLocalSeq -> ->
       ....
    1 ->
       ....
    _ ->
       ..... 
end,
```